src/test/scala/chiselTests/testers/VerilatorBackend.scala

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests.testers
+
+import chisel3.stage.ChiselGeneratorAnnotation
+import chisel3.stage.phases.{Convert, Emitter}
+import chiselTests.testers.TesterDriver.{AddImplicitTesterDirectory, cppToExe, executeExpectingSuccess, finishWrapper, verilogToCpp}
+import firrtl.AnnotationSeq
+import firrtl.options.{Dependency, PhaseManager, TargetDirAnnotation}
+import firrtl.stage.{FirrtlCircuitAnnotation, FirrtlStage}
```

Comment:
Can you put this in `src/test/scala/chisel3/testers` instead of in package `chiselTests`? By putting stuff in package `chiselTest` you're also moving `TestUtils` which I don't think you should.

Reply:
> Can you put this (and the other files being migrated into testing) in src/test/scala/chisel3/testers instead of in package chiselTests?

Yes, bug wondering why?

> By putting stuff in package chiselTest you're also moving TestUtils which I don't think you should.

I kept and deprecated original `TestUtils`, which I think this doesn't break anything?